art/attacks/iterative_method.py

```diff
@@ -48,99 +46,14 @@ def __init__(self, classifier, norm=np.inf, eps=.3, eps_step=0.1, max_iter=100,
         :type eps: `float`
         :param eps_step: Attack step size (input variation) at each iteration.
         :type eps_step: `float`
-        :param max_iter: The maximum number of iterations.
-        :type max_iter: `int`
         :param targeted: Should the attack target one specific class
         :type targeted: `bool`
-        :param num_random_init: Number of random initialisations within the epsilon ball. For num_random_init=0
-            starting at the original input.
+        :param num_random_init: Number of random initialisations within the epsilon ball. For random_init=0 starting at
```

Comment:
I think the `num_random_init` parameter should be removed.

Reply:
I have resolved this with I have resolved this with a38b34d8b5cfcbec46cee3b97fb6b9c98ebae283